src/components/tasks/TasksContent.tsx

```diff
@@ -85,7 +98,7 @@ export const TasksContent = () => {
 
     return (
         <div className={classNames.tasksContainer}>
-            <TabSection onSelect={onSelect} activeTab={activeTab} />
+            <TabSection onSelect={onSelect} activeTab={selectedTab as Tab} />
```

Comment:
- This looks like a potential bug, selectedTab value is `selected.toUpperCase()` where as the value of a variable of type `Tab` would be `selected` which are two different values.

- Also please try to avoid manual type casting as much as possible.

Reply:
For that to work, I will have to modify the internal component, as this has been drilled to 3 levels. and modifying everything would be out of the scope of this pr.